distribution/lib/Standard/Table/0.0.0-dev/src/Internal/In_Memory_Table_Implementation.enso

```diff
@@ -349,10 +348,10 @@ type In_Memory_Table_Implementation
             mask filter_column = from_java_table (this_table.java_table.filter (filter_column:In_Memory_Column).java_column)
             filter_condition = Filter_Condition.resolve_auto_scoped filter
             case filter_condition of
+                Filter_Condition.Predicate p -> mask (column.map p)
```

Comment:
- do I get it right that there is no `column.filter`, but
- we can create a `Column Boolean` and then use it as a mask for the original column?

Reply:
Yes that is how our filtering works in the table. We make a Boolean masking column to create a masked version.